Code/Editor/EditorFramework/Assets/Implementation/AssetBrowserFilter.cpp

```diff
@@ -61,6 +61,25 @@ void ezQtAssetBrowserFilter::SetTextFilter(const char* szText)
     return;
 
   m_sTextFilter = sCleanText;
+  // Clear uses search cache
+  m_bUsesSearchActive = false;
+  m_bTransitive = false;
+  m_uses.Clear();
+
+  const char* szUsesGuid = ezStringUtils::FindSubString_NoCase(szText, "uses:");
+  const char* szUses2Guid = ezStringUtils::FindSubString_NoCase(szText, "uses2:");
```

Comment:
How about "use:" and "ref:" instead of "uses" and "uses2" ?

Reply:
I was struggling with a good naming pattern here. The ezAssetCurator::FindAllUses finds all assets that have this asset as a transform dependency or reference it in addition to allowing to find indirect uses by creating the transitive hull.
I thought 'use' nicely generalized both refs and deps to the asset but couldn't think of a short form for the indirect case. I think ref would be misleading and wouldn't convey the indirect nuture. Granted, neither does uses2 but at least it isn't missleading.
Only in very rare cases will an asset be a transform dependency of another asset so we could get away with just calling the direct case ref: and the indirect case use: ?

Reply:
In my opinion "use" is the more direct thing. As in "this asset directly uses that asset", whereas "ref" is the more indirect thing, because asset A doesn't itself USE asset B, but still references it through another asset C.

Frankly I don't understand "Only in very rare cases will an asset be a transform dependency of another asset "

Reply:
ezAssetCurator understands and tracks **dependencies** as links between assets that are required to transform an asset. E.g. if A has a dependency on B, B needs to be successfully transformed for A to be able to be transformed.
**References** in the ezAssetCurator on the other hand are links between assets that are only relevant at runtime for the assest to function. E.g. if A has a reference to B, both A and B need to be successfully transformed in order for A to work at runtime.
Thus, I didn't want to reuse the same word for a different meaning which would be confusing.